packages/-build-infra/package.json

```diff
@@ -21,7 +23,20 @@
     "ember-cli-string-utils": "^1.1.0",
     "git-repo-info": "^2.0.0",
     "npm-git-info": "^1.0.3",
-    "silent-error": "^1.1.1"
+    "silent-error": "^1.1.1",
+    "glob": "^7.1.3",
+    "mocha": "^6.1.2",
+    "mocha-only-detector": "1.0.1",
+    "rimraf": "^2.6.2",
+    "rsvp": "^4.8.4",
+    "ember-cli-version-checker": "^3.1.2",
+    "ember-cli-blueprint-test-helpers": "^0.19.1"
+  },
+  "devDependencies": {
+    "broccoli-test-helper": "*",
+    "co": "*",
```

Comment:
Is this still needed? Since we are Node 8+ now we should make an issue to refactor our broccoli tests to async/await...

Reply:
https://github.com/emberjs/data/issues/6110